app/controllers/all_trips_controller.rb

```diff
@@ -0,0 +1,5 @@
+class AllTripsController < ApplicationController
+  def index
+    @trips = Trip.all.by_start_date
```

Comment:
Probably want to implement pagination here at some point. Maybe in another PR?

Reply:
Done in sha: `37f5010`